tests/console/vsftpd.pm

```diff
@@ -45,7 +46,13 @@ sub run {
     assert_script_run 'cd /tmp';
     assert_script_run 'wget ' . data_url('qam/vsftpd.tar.gz');
     assert_script_run 'tar xzfv vsftpd.tar.gz';
-    assert_script_run 'bash run.sh |& tee run.log', 300;
+    # Remove the cases with anonymous user in run_s390x because of bsc#1176813
+    if ($arch eq 's390x') {
+        record_soft_failure 'bsc#1176813 - vsftpd: security: one_process_model needs a better OS for the anonymous user scenarios';
+        assert_script_run 'bash run_s390x.sh |& tee run.log', 300;
```

Comment:
I doubt this will allow the next test to continue and would still mark them as failed, maybe script_run is a better choice, and reverse the condition:

soft fail if the runsh fails, die if the rush didn't fail (so we know that the test is passing, and can remove the soft failure).

Also I see that you've changed something in the data file, but do not explain the change anywhere that's visible.

Reply:
I see what you say, but I wanted to keep the scenarios of the test suite that pass, to continue running. These changes are located in the vsftpd tarball, which will be soon visible in this PR (I have marked it as WIP ;) )